wp-content/themes/core/components/blocks/gallery_grid/css/dialog.pcss

```diff
@@ -0,0 +1,268 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Dialog
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-dialog {
+
+    .swiper-wrapper {
+        align-items: center;
+        padding: 20px 0;
+    }
+
+    .swiper-slide {
+        text-align: center;
+    }
+
+    .swiper-pagination-fraction,
+    .swiper-pagination-custom,
+    .swiper-container-horizontal > .swiper-pagination-bullets {
+        display: block;
+        margin-top: 10px;
+        position: static;
+    }
+
+    .swiper-container-autoheight,
+    .swiper-container-autoheight .swiper-slide {
+        text-align: center;
+    }
+
+    .site-panel--gallery__col {
+        padding-bottom: 30px;
```

Comment:
Does this class exist in the blocks/components system? If it does, it should be renamed.